packages/nextjs/src/app/logout.ts

```diff
@@ -31,7 +31,7 @@ import { getPublicUrl } from "./utils"
 export async function getLogoutFlow({
   returnTo,
 }: { returnTo?: string } = {}): Promise<LogoutFlow> {
-  const h = await headers()
+  const h = headers()
```

Comment:
I think this was changed in next 15 and the new version requires an await (or the other way around). Please revert for now, I have it on my list to fix this issue in all places.

(Also if you do change stuff in both the elements-react directory and the next js directory, the commit will appear in the changelog for both. In this case with a totally unrelated message for next js.)

Reply:
Ah this is just a chore change, so it wouldn't appear in the changelog but still, a good idea to split most chnages unless strictly required.

Reply:
my bad that lust have crept in due to some IDE warning

```suggestion
  const h = await headers()
```

Reply:
reverted